gravitee-am-ui/src/app/domain/applications/application/advanced/oauth2/grantFlows/application-grant-flows.component.html

```diff
@@ -121,6 +121,30 @@ <h5>Advanced MTLS Client authentication</h5>
         </div>
       </div>
 
+      <div class="gv-form-section" *ngIf="applicationOauthSettings.tokenEndpointAuthMethod === 'private_key_jwt'">
```

Comment:
Should we do it also for `client_secret_jwt` ?

Reply:
By looking to the code, I think we are using the `client_secret` as an HMAC key to sign and verify the `client_secret_jwt`.